src/content-script/worker.ts

```diff
@@ -93,10 +96,12 @@ export class Worker {
       const scheduleAppointment = new Scheduler(params, userVisit);
       const res = await scheduleAppointment.handle(task);
       if (res.isDone) {
-        await this.stop();
-      } else {
-        this.settingAppointment = false;
+        await this.stop(res.status);
```

Comment:
If `isDone: true` it means that you never send status and it'll be undefined.
I would just call it without params, or am I missing something?

Reply:
It was sent in this.stop. But following your correct comment, I made it more explicit